backend.py

```diff
@@ -105,10 +105,15 @@ def getPrepSequence(ans, interactive):
     seq = [
         Task(util.getUUID, As(ans), ['installation-uuid']),
         Task(util.getUUID, As(ans), ['control-domain-uuid']),
-        Task(util.randomLabelStr, As(ans), ['disk-label-suffix']),
+        Task(util.randomLabelStr, As(ans), ['disk-label-suffix'])]
+
+    if ans['install-type'] == INSTALL_TYPE_FRESH and ans['swraid']:
+        seq.append(Task(setupSWRAIDDevice, A(ans, 'primary-disk', 'physical-disks'), []))
```

Comment:
I think this needs to move to after `removeBlockingVGs()` otherwise there may be blocking VGs that prevent creating the RAID device.